docs/content/docs/architecture/Modules.md

```diff
@@ -3,6 +3,104 @@ title: Modules
 type: docs
 ---
 
+# Modules
+
 {{<avito page>}}
 
-TBD: http://links.k.avito.ru/cfxFKKAg (internal)
+This page describes common structure of modules.
+
+## Modules grouping
+
+### By application
+
+On the top-level all modules are grouped by application:
+
+{{<mermaid>}}
+flowchart TB
+    subgraph avito-app-modules[ ]
+        avito-->avito-modules[...]
+        demo-app-->avito-modules
+    end
+    subgraph domofond-app-modules[ ]
+        domofond-->domofond-modules[...]
+    end
+    subgraph common-modules[ ]
+        common
+    end
+    avito-modules-->common-modules
+    domofond-modules-->common-modules
+{{</mermaid>}}
```

Comment:
![Screenshot from 2020-06-23 14-37-46](https://user-images.githubusercontent.com/1104540/85399271-35f52400-b55f-11ea-91c7-0857d2c6a213.png)

Reply:
I've tried different variants and decided to simplify as much a possible. It helps to avoid unimportant details.